radiant_mlhub/models.py

```diff
@@ -181,6 +192,21 @@ def registry_url(self) -> Optional[str]:
 
         return f'https://registry.mlhub.earth/{doi}'
 
+    @property
+    def archive_size(self) -> Optional[int]:
```

Comment:
Would it be useful to add an optional parameter which would return the size in a human readable format? i.e. "56.7 MB"

Could also be added as a separate method since it would return a string instead of an integer

Reply:
That's a good point. We could also have this return a `float` that represents the size in MB. That's probably a more useful unit for people anyway, and if folks are checking to see if they have enough space that level of rounding shouldn't make a big difference.

Reply:
I could also have the units be an optional argument, where `units` would be restricted to one of `"B"`, `"KB"`, `"MB"`, or `"GB"`.

```suggestion
    def archive_size(self, units: str = 'MB') -> Optional[float]:
```

Reply:
Not a fan of simply returning the size in MB, our archives range in size from 190 KB to 230 GB so you'd have a value ranging from 0.19 MB to 230000 MB

Reply:
> Not a fan of simply returning the size in MB, our archives range in size from 190 KB to 230 GB so you'd have a value ranging from 0.19 MB to 230000 MB

That's a good point. Maybe a separate property that automatically figures out the most appropriate units would be best then.

Reply:
I see two different use cases where a user would need to know the file size

> > Not a fan of simply returning the size in MB, our archives range in size from 190 KB to 230 GB so you'd have a value ranging from 0.19 MB to 230000 MB
> 
> That's a good point. Maybe a separate property that automatically figures out the most appropriate units would be best then.

I like that solution best. `archive_size` can be used for when you need to know exactly how many bytes the archive will take up, and `archive_size_readable` or something like that which can be used to quickly get a sense of the size before committing to downloading

Reply:
Since there might be additional useful information that a user would want to print, this might be best as part of a more general `info` property that prints a human-readable summary of the `Collection`; I opened #45 to track that feature. 

Since the `int` response will still be useful for programmatically checking available space, I'll leave this method in as-is for now.